### DISCUSSION TITLE
Deployment on Digital Ocean

### DISCUSSION THREAD

**Discussion by aforsythe** [marimo-team/marimo#607] [Q&A] [answered]
Wonderful tool! 

Sorry for the somewhat specific question ... 
I've created an app that I would deploy on Digital Ocean's App Platform.  Is that possible?

Thanks

**Comment by mscolnick** [comment, marked as answer]
hey @aforsythe, I will be posting a short blog write-up soon on how to deploy marimo apps via Docker. It's a short write up b/c its quote simple - since marimo is pure Python, there is no magic. Just install your requirements (e.g. `pip install requirements.txt`) and `marimo run app --port=8080`.

Let me know if you need help writing a Dockerfile, but myself and others in the discord can help out.

**Reply by aforsythe** [reply]
Thanks.  Yes this seems much less complicated than I'd initially anticipated, but a write up could be valuable. Just another random thought. It might be helpful to have a terminal and file browser in Marimo for the case where you're running it out of a docker container.

**Reply by aforsythe** [reply]
@mscolnick where is the blog btw?

**Reply by mscolnick** [reply]
hey @aforsythe, we added docs for deploying with Docker: https://docs.marimo.io/guides/deploying/deploying_docker.html

We don't have specific platform instructions, but if you deploy on Digital Ocean - we would appreciate the contribution for instructions

**Reply by akshayka** [reply]
@aforsythe the blog is here: https://marimo.io/blog/deploying-with-docker
